gems/kaminari-core/1.2/kaminari-core.rbs

```diff
@@ -0,0 +1,38 @@
+module Kaminari
+  def self.config: () -> Kaminari::Config
+  def self.configure: () { (Kaminari::Config) -> void } -> void
+
+  class Config
+    public attr_accessor default_per_page: Integer
+    public attr_accessor max_per_page: Integer?
+    public attr_accessor window: Integer
+    public attr_accessor outer_window: Integer
+    public attr_accessor left: Integer
+    public attr_accessor right: Integer
+    public attr_accessor page_method_name: Symbol | String | :page
+    public attr_accessor max_pages: Integer?
+    public attr_accessor params_on_first_page: bool
+
+    public attr_writer param_name: Symbol | String | :page | untyped
```

Comment:
`param_name` attribute accepts an object that receives `call` method.
How about defining an interface?

Reply:
Great idea! Thank you! I fixed it. 844cb848f7345fab7bf32ba298ccb78afc717acc